behaviour/graql/language/define.feature

```diff
@@ -354,6 +354,46 @@ Feature: Graql Define Query
     Then the integrity is validated
 
 
+  Scenario: define entity type with 'value' throws
+    Then graql define throws
+      """
+      define cream sub entity, value double;
+      """
+
+
+  Scenario: define type with a 'when' block throws
+    Then graql define throws
+      """
+      define gorilla sub entity, when { $x isa gorilla; };
+      """
+
+
+  Scenario: define type with a 'then' block throws
+    Then graql define throws
+      """
+      define godzilla sub entity, then { $x isa godzilla; };
+      """
+
+
+  Scenario: attempt to define a thing with 'isa' throws
+    Then graql define throws
+      """
+      define $p isa person;
+      """
+
+
+  Scenario: add attribute instance to thing in 'define' query throws
+    Then graql define throws
+      """
+      define $p has name "Loch Ness Monster";
+      """
+
+
+  @ignore
+  # TODO: re-enable when writing a variable in a 'define' is forbidden
```

Comment:
i think this may make a conflict with your other PR, heads up

Reply:
Seems ok to merge now